### PR TITLE
export Extents.extent

### PR DESCRIPTION
**Pull request by alex-s-gardner**
export Extents.extent. Addresses #757

### CODE REVIEW

File: src/Rasters.jl [hunk 1]
```diff
@@ -44,6 +44,7 @@ using RecipesBase: @recipe, @series
 using Base: tail, @propagate_inbounds
 
 import GeoInterface: crs
+import Extents: extent
```
Comment (rafaqz) [inline src/Rasters.jl:47]:
```suggestion
import Extents: Extent, extent
```

File: src/Rasters.jl [hunk 2]
```diff
@@ -64,7 +65,7 @@ export missingval, boolmask, missingmask, replace_missing, replace_missing!,
        coverage, coverage!, setcrs, setmappedcrs, smapseries, cellsize
 export crs, mappedcrs, mappedindex, mappedbounds, projectedindex, projectedbounds
 export reproject, convertlookup
-
+export extent
```
Comment (rafaqz) [inline src/Rasters.jl:68]:
```suggestion
export Extent, extent
```

**Review by asinghvi17 — APPROVED** [review]
Bump, this looks good to me?

**Comment by rafaqz** [comment]
Uggg probably DD should have the same export too

**Comment by tiemvanderdeure** [comment]
Big fan of this one, I've written `Rasters.Extents.extent` way to many times

**Pull request merged**